embc-app/ClientApp/src/app/volunteer/components/valid-from-to/valid-from-to.component.html

```diff
@@ -1,36 +1,46 @@
 <div class="row">
-  <div class="col-8">
+  <div class="col">
     <span class="h3">Valid From</span>
     <div class="row">
-      <div class="col-6">
-        <label for="validFromDate">Date</label>
-        <input type="text" id="validFromDate" class="form-control" [value]="validFrom">
-        <span class="invalid-feedback">Please enter a valid date.</span>
+      <div class="col">
+        <div class="input-group mb-0">
+          <label for="validFromDate_{{id}}">Date</label>
+          <input class="form-control" placeholder="yyyy-mm-dd" name="validFromDate_{{id}}" ngbDatepicker #fromDate="ngbDatepicker" [firstDayOfWeek]="7" navigation="arrows" [(ngModel)]="wrdForm.fromDate" (click)="fromDate.close();" (ngModelChange)="updateDisplay()" required aria-required="true">
```

Comment:
I don't know if you need the 'name' property, but for sure you need an 'id' property that the label refers to.